runners/flink/src/main/java/org/apache/beam/runners/flink/translation/functions/FlinkExecutableStageFunction.java

```diff
@@ -163,7 +194,70 @@ public void accept(WindowedValue<?> input) throws Exception {
         SdkHarnessClient.RemoteOutputReceiver<?>> receiverMap =
         receiverBuilder.build();
 
-    try (SdkHarnessClient.ActiveBundle<InputT> bundle = processor.newBundle(receiverMap)) {
+    StateRequestHandler stateRequestHandler = new StateRequestHandler() {
+      @Override
+      public CompletionStage<Builder> handle(StateRequest request) throws Exception {
+        if (request.getStateKey().getTypeCase() != TypeCase.MULTIMAP_SIDE_INPUT) {
+          throw new UnsupportedOperationException(
+              "This handler can only respond to MULTIMAP_SIDE_INPUT request.");
+        }
+        MultimapSideInput multimapSideInput = request.getStateKey().getMultimapSideInput();
+
+        String globalPCollectionName =
+            payload.getSideInputsOrThrow(multimapSideInput.getSideInputId());
+
+        List<Object> broadcastVariable = getRuntimeContext()
+            .getBroadcastVariable(globalPCollectionName);
+
+        PCollection pCollection = components.getPcollectionsOrThrow(globalPCollectionName);
+
+        String windowingStrategyId = pCollection.getWindowingStrategyId();
+        String windowCoderId =
+            components.getWindowingStrategiesOrThrow(windowingStrategyId).getWindowCoderId();
+
+        // TODO: This is the wrong place to hand-construct a coder, maybe
+        RunnerApi.Coder windowedValueCoder =
+            RunnerApi.Coder.newBuilder()
+                .addComponentCoderIds(pCollection.getCoderId())
+                .addComponentCoderIds(windowCoderId)
+                .setSpec(
+                    SdkFunctionSpec.newBuilder()
+                        .setSpec(
+                            FunctionSpec.newBuilder()
+                                .setUrn(validateCommonUrn("beam:coder:windowed_value:v1"))))
+                .build();
+
+        Coder javaCoder = CoderTranslation
+            .fromProto(windowedValueCoder, RehydratedComponents.forComponents(components));
+
+        // TODO: we wouldn't have to do this if the harness didn't always
+        // TODO: expect a KV<Void, T> as input for a side input, currently the key field
```

Comment:
We don't usually use continued TODO markers like this, but I can see why it might appear better in your IDE/editor this way. 😉